osf/models/node.py

```diff
@@ -1782,7 +1793,12 @@ def use_as_template(self, auth, changes=None, top_level=True, parent=None):
         new.save(suppress_log=True)
 
         # Need to call this after save for the notifications to be created with the _primary_key
-        project_signals.contributor_added.send(new, contributor=auth.user, auth=auth, email_template='false')
```

Comment:
@Johnetordoff Check if we are doubling up here to, but I think this stays, hard to claim a notification defined eslewhere is more valid.